From 11b61336930ba392b88e017d6b7cb135cd2a6068 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Thu, 6 Aug 2020 11:09:47 +0800
Subject: [PATCH] Test case7 for delete driver only located in dsc

Description: Delete the Driver (MdeModulePkg\Universal\DevicePathDxe\DevicePathDxe.inf) which only located in dsc component.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 8c881de93b..d4f38537c8 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -166,7 +166,7 @@
   }
   FatPkg/EnhancedFatDxe/Fat.inf
   TestPkg/AcpiTables/AcpiTables.inf
-  MdeModulePkg\Universal\DevicePathDxe\DevicePathDxe.inf
+  #MdeModulePkg\Universal\DevicePathDxe\DevicePathDxe.inf
 
 
 
-- 
2.27.0.windows.1

